esque/cli/commands.py

```diff
@@ -432,25 +433,41 @@ def edit_offsets(state: State, consumer_id: str, topic_name: str):
 
 
 @create.command("consumergroup")
-@click.argument("consumergroup-id", callback=fallback_to_stdin, required=True, type=click.STRING)
-@click.argument("topic", callback=fallback_to_stdin, required=True, type=click.STRING)
-@click.option("-o", "--offset", required=False, type=click.INT, help="Set offset(default to 0)")
-@click.option("-p", "--partition", required=False, type=click.INT, help="Set partition(default to 0)")
+@click.argument("consumergroup-id", callback=fallback_to_stdin, required=True, type=click.STRING, nargs=1)
+@click.argument("topics", callback=fallback_to_stdin, required=True, type=click.STRING, nargs=-1)
 @default_options
-def create_consumer_group(state: State, consumergroup_id: str, topic: str, offset: int, partition: int):
-    """Create consumer group for topic"""
-    if not ensure_approval("Are you sure?", no_verify=state.no_verify):
-        click.echo("Aborted!")
+def create_consumer_group(state: State, consumergroup_id: str, topics: str):
+    """Create consumer group for several topics using format <topic_name>[partition]=offset"""
+    offset_pattern = re.compile(r".+=(?P<offset>\d+)$")
+    main_pattern = re.compile(r"^(?P<topic_name>.+)\[(?P<partition>\d+)\].*")
```

Comment:
You can use this pattern to catch all cases :slightly_smiling_face:
https://regex101.com/r/4ydsOB/1
Non-matched groups will just be `None`

Reply:
Thanks!
Done